### ISSUE TITLE
Social integrations brokein in 1.1.3

### ISSUE THREAD

**Issue by vmajor** [mautic/mautic#729]
Edited crosspost from community forum.

Social integration does not work in 1.1.3. There are no errors even in my newly set up php error log.

1. Twitter integration is broken - confirmed, bug scheduled to be fixed "in the next version"
 
2. Facebook integration is broken - unconfirmed, silent error - nothing logged anywhere and no output of any kind on the screen

3. Google+ is broken - unconfirmed, pulls up random profile that matches the users's first name, and does not give up regardless of what other Google+ account is entered into the Google+ social profile field. Even if the Google+ social profile field is empty it still pulls up that same random person that shares the first name with me (the lead).

**Comment by alanhartless** [comment]
Hi!

Twitter is fixed with #715.

For Facebook, it seems that they have shut down their API access to search by username for public information without explicit permission of the Facebook user.  It used to work but now returns an error that searching by username is not allowed.  We'll be looking into our options for this but for now, it seems that social profiling for Facebook is on hold.

We'll have to investigate the Google+ issue.  It's only supposed to search by email or Google+ plus profile username.

**Comment by cathalgarvey** [comment]
+1
Any idea when the twitter fix will be integrated and pushed to Mautic.com?

**Comment by dbhurley** [comment]
Hi!  The team at Allyde normally pushing regular rolling releases to Mautic.com based on the community codebase. Since they are running Mautic with additional add-ons built on top of it (AllydeMautic) they have a different testing setup that must be done before pushing a community fix or pull request to that infrastructure. However usually this happens quickly!

**Comment by cathalgarvey** [comment]
Great, because this is kind of a huge selling point for Mautic, and it's been broken since I started using it.

Are the test cases getting improved to stop this happening again? I'd offer to help if I spoke PHP, but I'm afraid I've next to no experience with it..

**Comment by vmajor** [comment]
...most integrations are broken. I also cannot get Mailchimp to work, and I cannot get Gmail to work. I gave up trying anything else. The whole experience with Mautic has been very underwhelming. The basic premise of "automation" is to reduce the rote workload and allow the users to focus on higher level outcomes and processes. But if the automation tool is so broken that it requires debugging of just about every "nice" feature, and thus redoing of any potential higher level business process, what is the point?

I will try to see how 1.1.4 behaves, if it is no better then I'll say bye to Mautic until perhaps version 2.x. I have no time or desire for beta testing.

**Comment by dbhurley** [comment]
Please make sure issue comments provide value to the issue either by providing additional information or stating an environment you're working in to help developers better resolve the issue.

Thanks everyone!

**Comment by vmajor** [comment]
Hi David, what you would like to know in terms of additional information? 

At a guess here is the PHP information:

PHP 5.6.4-4ubuntu6.2 (cli) (built: Jul  2 2015 15:18:51) 
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2014 Zend Technologies
    with Zend OPcache v7.0.4-dev, Copyright (c) 1999-2014, by Zend Technologies

I am using Lubuntu 15.04

What else would you like to know?

**Comment by dbhurley** [comment]
Awesome :+1: That is much appreciated. Since we have Twitter fixed and G+ is under investigation maybe let's look at some of the other issues that are not related to social media on a separate issue. I haven't heard of any issues with Mailchimp but I think you mentioned that. Can we start an issue on that and get some feedback on the error messages you received?

**Comment by vmajor** [comment]
OK, Mailchimp... that is a user error - mine.

Historically all our other integrations were using just a Mailchimp API key. Mautic however required a creation of an "App". I did not see a mention of this requirement anywhere (or I missed it) so I went and created an "App" and changed my integration details in Mautic. It now appears to work.

**Comment by vmajor** [comment]
Let's do Google next. This one is an "easy" one. I will open a new thread.

**Comment by alanhartless** [comment]
The twitter error has been fixed.
